ksonnet/dkube/deployment.libsonnet

```diff
@@ -3,7 +3,7 @@
 	$.parts(params.namespace).logstash(params.tag, params.logstashImage, params.dkubeDockerSecret),
 	$.parts(params.namespace).dkubeEtcd(params.tag, params.dkubePVC),
 	$.parts(params.namespace).dfabProxy(params.tag,params.dfabProxyImage, params.dkubeDockerSecret),
-	$.parts(params.namespace).dkubeWatcher(params.tag, params.dkubeWatcherImage, params.dkubeDockerSecret),
+	$.parts(params.namespace).dkubeWatcher(params.tag, params.dkubeWatcherImage, params.dkubeDockerSecret, params.isRdmaEnabled),
```

Comment:
Did you change prototypes/all.jsonnet as well? It still has the foll.

_// @optionalParam rdmaEnabled string false RDMA enable flag_